src/app/public/modules/vertical-tabset/vertical-tab.component.ts

```diff
@@ -112,6 +112,8 @@ export class SkyVerticalTabComponent implements OnInit, OnDestroy {
 
   public index: number;
 
+  public contentRendered: boolean;
```

Comment:
Check your alphabetical ordering of properties.

Reply:
Done